util/prepare_environment/LICENSE

```diff
@@ -0,0 +1,38 @@
+
+Android swift compiler and toolchain for Ubuntu 16.04
+
+Updates will be posted here: http://johnholdsworth.com/android_toolchain.tgz
+
+Other resources
+
+Use the enclosed script setup_ubuntu_client.sh to setup your linux environment.
+
+Check out the [SwiftAndroid](https://github.com/SwiftAndroid) and
+[SwiftJava](https://github.com/SwiftJava/swift-android-samples)
+github projects for starter Android applications and resources.
+
+This binary package contains software built from the following repositories:
+
+https://github.com/curl/curl
+
+git://git.gnome.org/libxml2 or
+https://github.com/android/platform_external_libxml2
+
+https://github.com/SwiftAndroid/libiconv-libicu-android
+
+https://github.com/apple/swift
+
+https://github.com/apple/swift-corelibs-libdispatch
+
+https://github.com/apple/swift-corelibs-foundation
+
+Built using a fork of the scripts in:
+https://github.com/gonzalolarralde/swifty-robot-environment
```

Comment:
Hi @ephemer, I’m confused by this comment. I should take acknowledgement of @gonzalolarralde’s scripts out of any binary release I that I do?